lib/seam/routes/clients/events.rb

```diff
@@ -2,12 +2,12 @@
 
 module Seam
   module Clients
-    class Events < BaseClient
+    class SeamEvents < BaseClient
```

Comment:
This can stay `Events`. It's only the `Event` class we need to worry about.

Reply:
Gotcha. Should we change it back on [Python SDK](https://github.com/seamapi/python/blob/370520b1ab8ea88a4375f8bf69215df78d004f3c/seam/routes/events.py#L6) as well?

Reply:
Yes, sure. The class isn't really part of the public API of the SDK so it should be ok to rename.

Reply:
c974a700b5004432bce61c824f384a4a3b993108